doc/rst/developer/bestPractices/Unstable.rst

```diff
@@ -0,0 +1,42 @@
+.. _best-practices-unstable:
+
+Unstable Features
+=================
+
+When a feature's current interface is not meeting demands but it is uncertain
```

Comment:
Maybe "When it is known that a feature is likely to require discussion or changes..."

I don't really like "meeting demands" as wording here.